adds security note (#563)
logzio/logz-docs: _source/user-guide/integrations/custom-endpoints.md

<div class="tasklist">

> ###### Add the endpoint

Author: ```suggestion
##### Add the endpoint
```

To add a new custom endpoint, click **Add endpoint**.
Specify the URL, method, and headers.
For POST/PUT methods, you can configure the message body, if relevant.

> ###### Ports & security considerations
> 
> It is best to avoid opening a port in your firewall to allow access for custom endpoints.
> 
> The Logz.io IP range is used by all customers, so it should be used with caution.
> If you do open a port for a Logz.io custom endpoint, follow these best practices:
> 
> * Make sure that you add a token in your custom payload and verify that the token exists in the endpoint.
> * Allow external access only to this specific endpoint.
> 
> Alerts can only be sent on ports 80 & 443.
> If you accidently set a custom port for an alert endpoint, the alert will not be sent.
> {:.info-box.important}

Author: ```suggestion
###### Ports & security considerations

Alerts can only be sent on ports 80 & 443.
If you accidently set a custom port for an alert endpoint, the alert will not be sent.

It is best to avoid opening a port in your firewall to allow access for Logz.io custom endpoints.
This is because the Logz.io IP range is used by all customers and should be treated with caution.

If you decide to open a port on your firewall for a Logz.io custom endpoint, follow these best practices:

* Make sure to add a token in your custom payload and verify that the token exists in the endpoint.
* Allow external access only to this specific endpoint.

```

<div class="tasklist">

##### Add the endpoint

To add a new custom endpoint, click **Add endpoint**.
Specify the URL, method, and headers.
For POST/PUT methods, you can configure the message body, if relevant.

###### Ports & security considerations

Alerts can only be sent on ports 80 & 443.
If you accidently set a custom port for an alert endpoint, the alert will not be sent.

> 
> It is best to avoid opening a port in your firewall to allow access for Logz.io custom endpoints.
> This is because the Logz.io IP range is used by all customers and should be treated with caution.
> 
> If you decide to open a port on your firewall for a Logz.io custom endpoint, follow these best practices:
> 
> * Make sure to add a token in your custom payload and verify that the token exists in the endpoint.
> * Allow external access only to this specific endpoint.
> 

Author: ```suggestion

It is best to avoid opening a port in your firewall to allow access for Logz.io custom endpoints. 
If you decide to open a port on your firewall for a Logz.io custom endpoint, follow these best practices:

* Make sure to add a token in your custom payload and verify that the token exists in the endpoint.
* Allow external access only to this specific endpoint.

Here's an example of a body message for a POST endpoint that has a token for added security in the JSON payload. This simply means adding a single JSON field:
```
{
    “token”: “{{SomeSecurityToken}}”,
    "alert_title": "{{alert_title}}",
    "alert_description": "{{alert_description}}",
    "alert_severity": "{{alert_severity}}",
    "alert_event_samples": "{{alert_samples}}"
}
```

The Logz.io IP range is used by all customers and should be treated with caution.
{:.info-box.important}

```
